src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/step/ValidateClusterNameStep.java

```diff
@@ -20,13 +20,13 @@ public void execute(ClusterConfiguration config)
         if (clusterName == null)
         {
             throw new ElasticsearchSetupException(String.format(
-                    "Please provide an alphanumeric cluster name."));
+                    "Please provide a cluster name."));
         }
-        
-        if (clusterName.matches("[a-zA-Z0-9]+") == false)
+
+        if (clusterName.matches("[a-zA-Z0-9.-]+") == false)
```

Comment:
`.` means any character; make sure you escape it: `\.`

Reply:
Actually `.ˋ is sufficient because of ˋ[ ]ˋ
The unit test is throwing exception web there is a ˋ‘ˋ in the name